torchaudio/functional.py

```diff
@@ -431,6 +496,7 @@ def create_fb_matrix(
         f_max (float): Maximum frequency (Hz)
         n_mels (int): Number of mel filterbanks
         sample_rate (int): Sample rate of the audio waveform
+        htk (bool): Use HTK formula instead of Slaney
```

Comment:
Should this instead be a flag that allows the user to pass a selection (e.g. "Slaney" or "HTK") so that we could add more formulas later on (if that could come up)?

Reply:
The `htk` boolean flag is a convention from librosa. Someone had also suggested `mel_scale="htk"|"slaney"`, see [here](https://github.com/pytorch/audio/pull/426#discussion_r376505782).

Reply:
Is there a reason htk (bool) is preferred over the mel_scale flag?

Reply:
I don't see other reasons. I prefer `mel_scale` personally as it is more explicit about which convention is being used.

Reply:
Changed to `mel_scale`